### ISSUE TITLE
Remove 'h5' element from top-term markup

### ISSUE THREAD

**Issue by benlk** [INN/Largo#1786]
Having headings out of order messes with the document outline, according to various tests.

> To solve this violation, you need to:
> Fix the following:
> 
> Heading order invalid

- [ ] replace markup in the `largo_maybe_top_term` function with a ... `span`, I guess.
- [ ] replace `h5.top-term` and `h5.top-tag` styles with `.top-term` across the board, and check visual appearances

---- 
This issue was identified during research on https://github.com/INN/largo/issues/1653 using the AXE browser plugin.
